.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: node_js
 
 node_js:
-  - "lts/argon"
-  - "lts/boron"
-  - "node"
+  - "8"
```

Comment:
You may want to follow hapi's lead in the versions that should be tested against. See here: https://github.com/hapijs/hapi/blob/d3bbf3ac5955bcfc7ff6dc770afff79ca8c95bd8/.travis.yml#L3-L6